sunpy/visualization/wcsaxes_compat.py

```diff
@@ -106,8 +106,10 @@ def default_wcs_grid(axes):
 
 
 @u.quantity_input
-def wcsaxes_heliographic_overlay(axes, grid_spacing: u.deg = 10*u.deg, annotate=True,
-                                 obstime=None, rsun=None, **kwargs):
+def wcsaxes_heliographic_overlay(axes, system='stonyhurst',
```

Comment:
Is putting the `system` arg here a breaking change for anyone who's using positional args? Or is this private API? (Should we use [keyword-only arguments](https://www.python.org/dev/peps/pep-3102/) for things like this?)

Reply:
Yes, good catch. I'll move it to the end.